charts/opa-kube-mgmt/values.schema.json

```diff
@@ -31,6 +31,21 @@
         "annotations": {"type": "object", "additionalProperties": {"type": "string"}, "default": {}},
         "name": {"type": ["string", "null"], "default": null}
       }
+    },
+    "service": {
+      "type": "object",
+      "properties": {
+        "annotations": {
+          "type": "object",
+          "additionalProperties": {"type": "string"},
+          "default": {}
+        },
+        "trafficDistribution": {
+          "type": ["string"],
```

Comment:
I assume 
* `type` should be `["null", "string"]`
* `enum` should not contain empty string value `""`

Reply:
yes, it makes more sense. Thanks for the input. Updated in `19fa78cb`